src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -4,7 +4,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-from knack.help_files import helps  # pylint: disable=unused-import
+from knack.help_files import helps  # pylint: disable=import-error
```

Comment:
Why disable import-error

Reply:
> I prefer to use None in if statement explicitly. Like `if var is None` or `if var is not None`. Be careful.

Ok - I will double check my code. I attempted to be careful to only use `if X, or if not X` in scenarios where I was comfortable checking for all truthy / falsy and where it made my code cleaner (e.g. avoiding having to do if not none and if not ""). But I will double check to make sure I didn't slip up on any of them.